apps/docs/src/content/docs/contributing/labrinth.md

```diff
@@ -62,18 +62,18 @@ During development, you might notice that changes made directly to entities in t
 
 #### Basic configuration
 
-`DEBUG`: Whether debugging tools should be enabled  
-`RUST_LOG`: Specifies what information to log, from rust's [`env-logger`](https://github.com/env-logger-rs/env_logger); a reasonable default is `info,sqlx::query=warn`  
-`SITE_URL`: The main URL to be used for CORS  
-`CDN_URL`: The publicly accessible base URL for files uploaded to the CDN  
-`MODERATION_DISCORD_WEBHOOK`: The URL for a Discord webhook where projects pending approval will be sent  
-`CLOUDFLARE_INTEGRATION`: Whether labrinth should integrate with Cloudflare's spam protection  
-`DATABASE_URL`: The URL for the PostgreSQL database  
-`DATABASE_MIN_CONNECTIONS`: The minimum number of concurrent connections allowed to the database at the same time  
-`DATABASE_MAX_CONNECTIONS`: The maximum number of concurrent connections allowed to the database at the same time  
-`MEILISEARCH_ADDR`: The URL for the MeiliSearch instance used for search  
-`MEILISEARCH_KEY`: The name that MeiliSearch is given  
-`BIND_ADDR`: The bind address for the server. Supports both IPv4 and IPv6  
+`DEBUG`: Whether debugging tools should be enabled
+`RUST_LOG`: Specifies what information to log, from rust's [`env-logger`](https://github.com/env-logger-rs/env_logger); a reasonable default is `info,sqlx::query=warn`
+`SITE_URL`: The main URL to be used for CORS
+`CDN_URL`: The publicly accessible base URL for files uploaded to the CDN
+`MODERATION_DISCORD_WEBHOOK`: The URL for a Discord webhook where projects pending approval will be sent
+`CLOUDFLARE_INTEGRATION`: Whether Labrinth should integrate with Cloudflare's spam protection
```

Comment:
Isn't labrinth typically referred to in lower case? Also line 117.

Reply:
I've switched to lowercase now. See https://github.com/modrinth/code/pull/4132#discussion_r2264638819 for the rationale.